sys/net/gnrc/application_layer/dhcpv6/client.c

```diff
@@ -85,6 +85,9 @@ void dhcpv6_client_conf_prefix(unsigned iface, const ipv6_addr_t *pfx,
     gnrc_netif_t *netif = gnrc_netif_get_by_pid(iface);
     int idx = gnrc_netif_ipv6_add_prefix(netif, pfx, pfx_len, valid, pref);
     if (idx >= 0) {
+        /* start advertising subnet obtained via DHCPv6 */
+        gnrc_ipv6_nib_change_rtr_adv_iface(netif, true);
```

Comment:
As `dhcpv6_client_conf_prefix` is also used for IA_NA now, would be there be an additional check needed here? (For instance, that the prefix length is not equal to `IPV6_ADDR_BIT_LEN`?)

Reply:
(I hope reviewing makes sense by the way, I stumbled on this PR by chance and noticed that it is still open.)

Reply:
Good catch, fixed